public/scripts/extensions/memory/settings.html

```diff
@@ -14,8 +14,25 @@
                     <option value="main" data-i18n="ext_sum_main_api">Main API</option>
                     <option value="extras">Extras API (deprecated)</option>
                     <option value="webllm" data-i18n="ext_sum_webllm">WebLLM Extension</option>
+                    <option value="profile" data-i18n="ext_sum_profile">Connection Profile</option>
                 </select><br>
 
+                <!-- Connection Profile section (only visible when "profile" is selected) -->
+                <div data-summary-source="profile" style="display:none;">
+                    <div class="openai_restorable flex-container">
+                        <div class="title_restorable">
+                            <label>Connection Profile</label>
```

Comment:
```suggestion
                            <label data-i18n="Connection Profile">Connection Profile</label>
```
Help our translator friends.